src/NuGet.Clients/VsExtension/.vsixignore

```diff
@@ -4,7 +4,40 @@ Microsoft.CSharp.dll
 Microsoft.IdentityModel.Clients.ActiveDirectory.dll
 Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll
 Microsoft.ServiceBus.dll
+Microsoft.TeamFoundation.Build.Client.dll
```

Comment:
Numerous are introduced into the build via the NuGet package flow from VisualStudio14.Packages, and we have no need to include them in the VSIX.